src/qspdlog_model.cpp

```diff
@@ -114,3 +126,6 @@ QVariant QSpdLogModel::headerData(
 
     return QVariant();
 }
+void QSpdLogModel::setMaxEntries(std::optional<std::size_t> maxEntries) {
+    _maxEntries = maxEntries;
+}
```

Comment:
Move before the getter and keep the one-line margin between the function bodies.